token/wallet.tact

```diff
@@ -2,164 +2,120 @@
 
 include "common.tact";
 
-let STORAGE_MIN_CONSUMPTION = Coins.minitons(10000000); // 0.01 TON
-let GAS_CONSUMPTION = Coins.minitons(10000000); // 0.01 TON
-
-struct WalletState {
-    val balance: Coins;
-    val address: ContractAddress;
-    val master: ContractAddress;
-    val wallet_code: Cell;
-
-    @derive impl Serialize {}
-    @derive impl Deserialize {}
-}
-
-union InternalMsgBody {
-    case Transfer;
-    case InternalTransfer;
-    case Burn;
+let STORAGE_MIN_CONSUMPTION = Coins.militons(10); // 0.01 TON
```

Comment:
```suggestion
let STORAGE_MIN_CONSUMPTION = Coins.millitons(10); // 0.01 TON
```

Reply:
Fixed in [84b5764](https://github.com/tact-lang/tact-bikeshed/pull/5/commits/84b5764e9cd999aff5190635c032cdbc74c5c0f9)